aspnetcore/razor-pages/ui-class.md

```diff
@@ -221,3 +221,30 @@ Suppose *RazorUIClassLib/Pages/Shared* contains two partial files: *_Header.csht
   <partial name="_Footer">
 </body>
 ```
+
+## Create a Razor class library with static assets
+
+Razor class libraries (RCL) frequently require companion static assets that can be referenced by the consuming app of the RCL. ASP.NET Core allows creating RCLs that include static assets that are available to a consuming app.
```

Comment:
The plural is RCL in one place and RCLs in another place.  Recommend standardize on RCLs for plural.

 ASP.NET Core allows creating RCLs that include static assets -->  RCLs can include static assets

Reply:
Let's go with RCL. It's defined at the top of the topic and then used in many spots prior to adding the new content on this PR.